ccdproc/tests/test_ccddata.py

```diff
@@ -518,3 +518,6 @@ def test_wcs_attribute(ccd_data, tmpdir):
         assert k not in ccd_wcs_not_in_header.header
         # Every keyword in the WCS should be in the header of the HDU
         assert hdu.header[k] == wcs_header[k]
+
+def test_header(ccd_data):
+    ccd = CCDData(ccd_data, header={})
```

Comment:
Could we make this test a little stronger by defining a dictionary, e.g. `a={'observer': 'hubble'}`, passing that into `CCDData` as `header